lib/modules/manager/composer/artifacts.spec.ts

```diff
@@ -1183,4 +1183,26 @@ describe('modules/manager/composer/artifacts', () => {
       },
     ]);
   });
+
+  it('includes new dependency version in update command', async () => {
+    fs.readLocalFile.mockResolvedValueOnce('{}');
+    const execSnapshots = mockExecAll();
+    fs.readLocalFile.mockResolvedValueOnce('{}');
+    git.getRepoStatus.mockResolvedValueOnce(repoStatus);
+
+    expect(
+      await composer.updateArtifacts({
+        packageFileName: 'composer.json',
+        updatedDeps: [{ depName: 'foo', newVersion: '1.1.0' }],
```

Comment:
Can you also enhance the previous (existing) test to have new versions for _both_ updates and then update the matchObject for it?

Reply:
Sure!